pkg/controller/events/event_forwarder.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"context"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+type EventHandler struct {
+	client.Client
+	Scheme *runtime.Scheme
+}
+
+func (e *EventHandler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx)
+
+	event := &corev1.Event{}
+	if err := e.Get(ctx, req.NamespacedName, event); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	obj := &unstructured.Unstructured{}
+	obj.SetGroupVersionKind(schema.FromAPIVersionAndKind(event.InvolvedObject.APIVersion, event.InvolvedObject.Kind))
+	if err := e.Get(ctx, types.NamespacedName{Name: event.InvolvedObject.Name, Namespace: event.InvolvedObject.Namespace}, obj); err != nil {
+		return ctrl.Result{}, err
+	}
+	ef := obj.GetAnnotations()["vshn.net/forward-events-to"]
```

Comment:
in which place do we set this annotation? or is it meant to be set by user?

Reply:
It depends. For helm based setups, this should be set in the composition by setting the annotations of the objects in the `helmValues`. (for Redis eg: https://github.com/vshn/component-appcat/pull/258
For provider specific setups (eg. PostgreSQL) this should most likely be done in the provider.